samples/caf/README.rst

```diff
@@ -50,12 +50,13 @@ User interface
 Button 1:
   Changes the LED effect displayed on the LED to the next defined LED effect:
 
-  * The first default LED effect toggles the blinking of the LED, with the period set to 200 ms.
+  * The first default LED effect toggles the blinking of the LED, with the period set to 400 ms.
   * The second default LED effect toggles the breathing of the LED, with the period set to 400 ms.
-  * The third default LED effect toggles a looped effect where blinks with the period set to 200 ms and then turns off for 1000 ms.
+  * The third default LED effect toggles a looped effect where blinks with the period set to 400 ms and then turns off for 1000 ms.
```

Comment:
`... where LED blinks three times with period of 400 ms ....`?

Reply:
fixed